javascript/javascript2/week1/hmw2.js

```diff
@@ -0,0 +1,25 @@
+const danishString = ["Jeg har en blå bil"];
+const danishString2 = ["Blå grød med røde bær"];
+function danish(danishString) {
+  const string = danishString.join();
+  let count = 0;
+  let obj = {};
```

Comment:
Try to give your variables meaningful names. For example
```suggestion
  let danishLetters = {};
```